sass/materialize.scss

```diff
@@ -1,17 +1,16 @@
 @charset "UTF-8";
 
-// Mixins
-// @import "components/prefixer";
-@import "components/mixins";
-@import "components/color";
-
-// Variables;
+// 1. Variables
 @import "components/variables";
 
-// Reset
+// 2. Reset
 @import "components/normalize";
 
-// components
+// 3. Mixins
+@import "components/prefixer";
```

Comment:
@tomscholz is right https://github.com/Dogfalo/materialize/blob/e99b963a1e3bc8b50476cbc48de57d73194fdf4f/Gruntfile.js#L77-L103